public/static/locales/de/tenants.json

```diff
@@ -27,5 +27,9 @@
   "lacoqueta": {
     "title": "Bäume pflanzen mit La Coqueta",
     "description": "Zu sehen, wie ihre eigenen Kinder aufwachsen und sich zunehmend für die Klimakrise interessieren, hat Celia dazu inspiriert, ihre Marke als Plattform für positiven Wandel zu nutzen. Nachdem sie erfahren hatte, dass Bäume das wirtschaftlichste und effektivste Mittel sind, um CO2 zu binden und so mehr Zeit zu gewinnen, um die Treibhausgasemissionen auf Null zu reduzieren und die Klimakrise zu entschärfen, entschied sie sich, diese spannende Reise mit Plant-for-the-Planet anzutreten. Auch Sie können Teil dieser Reise werden und einige Bäume spenden - in Granada, der Heimatstadt von Celia, oder in anderen Gebieten auf der ganzen Welt."
+  },
+  "ulmpflanzt": {
+    "title": "Ulm-pflanzt ❤️ Baeume",
```

Comment:
"title": "Ulm pflanzt Baeume" would be preffered

Reply:
Because "Ulm pflanzt Baeume" means "Ulm plants trees".
And that makes "Ulm-plants (loves) trees" weird in context

Reply:
changed